terragrunt/aws/api/lambda.tf

```diff
@@ -44,3 +50,23 @@ resource "aws_cloudwatch_event_target" "trigger_api_lambda_to_rescan" {
   arn       = module.api.function_arn
   input     = jsonencode({ task = "assemblyline_resubmit_stale" })
 }
+
+# Update ClamAV virus database every 2 hours
+resource "aws_cloudwatch_event_rule" "clamav_update_avdefs" {
+  name                = "clamav-update-avdefs-${var.env}"
+  description         = "Updates ClamAV virus database every 2 hours"
+  schedule_expression = "rate(2 hour)"
+  is_enabled          = false # Disabled until we have a working solution
```

Comment:
any reason why we can't start updating the filestore?

Reply:
@wmoussa-cds Just waiting until the e2e solution is up and running since this PR only introduced downloading updates. https://github.com/cds-snc/scan-files/pull/102 will have the API that kicks off scans and enables the av updates.